config/nim.cfg

```diff
@@ -114,6 +114,8 @@ nimblepath="$home/.nimble/pkgs/"
     clang.options.linker = "-Wl,--as-needed -lnetwork"
     clang.cpp.options.linker = "-Wl,--as-needed -lnetwork"
     tcc.options.linker = "-Wl,--as-needed -lnetwork"
+  @elif genode:
+    discard
```

Comment:
The configuration syntax doesn't support ``discard``. But I wonder why it reports no error on the CIs... :-)

Reply:
Maybe the CI never causes evaluation here?

Reply:
indeed, that's because genode isn't tested in CI; otherwise it'd give:
```
Error: invalid command line option: '--discard'
```
as expected

Reply:
I've using these patches against v1.4.4, and this file has changed, so I didn't catch that.